Tests/Pcap++Test/Tests/TcpReassemblyTests.cpp

```diff
@@ -211,6 +268,47 @@ static bool tcpReassemblyTest(const std::vector<pcpp::RawPacket>& packetStream,
 }
 
 
+// ~~~~~~~~~~~~~~~~~~~
+// tcpReassemblyTestManuallyCloseConnOnMesgReady()
+// ~~~~~~~~~~~~~~~~~~~
+
+static bool tcpReassemblyTestManuallyCloseConnOnMesgReady(const std::vector<pcpp::RawPacket>& packetStream, TcpReassemblyMultipleConnStatsWithReassembly& results, bool monitorOpenCloseConns, bool closeConnsManually)
```

Comment:
I think we can make this method simpler - I don't think that `monitorOpenCloseConns` and `closeConnsManually` are needed...